views/beers.hbs

```diff
@@ -0,0 +1,6 @@
+<div class="beerlist">
+  {{#each beersFromApi}}
+    <img  class= 'imgb'src='{{image_url}}' alt='' />
```

Comment:
Wrap this items inside a div to create a card, like this:
```
<div class="beerlist">
  {{#each beersFromApi}}
  <div class="beerCard" >
    <img  class= 'imgb'src='{{image_url}}' alt='' />
    <p>{{name}}</p>
  </div>
  {{/each}}
</div>
```

then change on css the beerlist property to 
`flex-direction: row` 

and after that specify the width of your card

```
.beerlist {
    display: flex;
    flex-direction: row;
    flex-wrap: wrap;
    justify-content: center;
    align-items: center;
}
.beerCard {
    width: 25%
}

```